blog/2018/12/07/openapi/index.markdown

```diff
@@ -0,0 +1,169 @@
+# MetaCPAN, Mojolicious and OpenAPI
+
+During this years [meta::hack 3](http://www.olafalders.com/2018/11/21/metahack-3-wrap-report/), I was extremely fortunate to work with Joel Berger on integrating/documenting OpenAPI with the MetaCPAN API via Mojolicious.
+
+## What is it?
+OpenAPI is a specification for designing, documenting, validating and driving your RESTful API. It can be used to provide documentation to an existing API, or staring fresh.
+
+The OpenAPI Specification originated as the Swagger specification and was renamed to separate the API description format from the open source tooling. The specification moved into new GitHub repository, but did not change.
```

Comment:
add parentheticals to denote which is which: "the API description format (OpenAPI) from the open source tooling (Swagger)"